signer/contentsignaturepki/upload.go

```diff
@@ -26,21 +34,28 @@ func (s *ContentSigner) upload(data, name string) error {
 	}
 	switch parsedURL.Scheme {
 	case "s3":
-		return uploadToS3(data, name, parsedURL)
+		// Context is a required argument, but in our uses,
+		// LoadDefaultConfig pulls the necessary configuration
+		// from the environment.
+		cfg, err := config.LoadDefaultConfig(context.Background())
```

Comment:
Not for this PR and not asking for this to change because we're inheriting prior decisions here, but noting only so that future folks can see our thinking: these `LoadDefaultConfig` and `NewFromConfig`s are typically meant to happen at boot time, and that we do them over and over is a lil suspect. Not a big deal because we only do dozens of these a month, but not quite what's expected

Reply:
Yeah, I'm going to forego doing it as part of this, but seeing as https://mozilla-hub.atlassian.net/browse/AUT-182 will necessitate some changes to configuration, I may end up doing it in advance or as part of that work.